torch/csrc/utils/python_arg_parser.h

```diff
@@ -249,6 +250,15 @@ inline at::Tensor PythonArgs::tensor(int i) {
   return tensor_slow(i);
 }
 
+inline c10::optional<at::Tensor> PythonArgs::optionalTensor(int i) {
+  at::Tensor t = tensor(i);
+  if (t.defined()) {
```

Comment:
umm if we know the schema, can't we decide if we allow Nones or not?  This looks really sketchy, as the only difference between optionalTensor and tensor is how we map `Nones`.